src/routes/choice-exercises/components/SingleChoiceExercise.tsx

```diff
@@ -119,7 +119,14 @@ const ChoiceExerciseScreen = ({
         selectedAnswer={selectedAnswer}
       />
       <ButtonContainer>
-        {selectedAnswer !== null && (
+        {selectedAnswer === null ? (
+          <Button onPress={() => {}}>
```

Comment:
Have you forgotten something here? The button should do something. Maybe the ticket was not clear. It should work like the try-later button in the write exercise, the word you skipped when clicking the try-later button, should be shown again as the last word of the exercise.

Reply:
Was not sure if it's part of the issue. Will update as soon as LUN-114 (fixing the try-later in exercise 4) is finished.

Reply:
Maybe it makes sense, if you close this PR and start with the current main to avoid merge conflicts.

Reply:
I was planning on just pulling from main as soon as LUN-114 is finished before I continue implementing for this issue.